metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -640,59 +641,62 @@ def get_parser(
         connector_manifest: ConnectorManifest,
     ) -> DebeziumParser:
         connector_class = connector_manifest.config.get("connector.class", "")
+        # debezium version convention : x.y.z.{Final, Alpha, Beta, CR}
+        connector_version = int(self.version.split('.')[0])
```

Comment:
Is version always present ? Can we handle for missing version (i.e. version=None)

Reply:
newer version of debezium(2.1) is released
https://debezium.io/releases/

Reply:
here is rest api return values sample 
```json
[
  {
    "class": "com.wepay.kafka.connect.bigquery.BigQuerySinkConnector",
    "type": "sink",
    "version": "unknown"
  },
  {
    "class": "io.confluent.kafka.connect.datagen.DatagenConnector",
    "type": "source",
    "version": "null"
  },
  {
    "class": "io.debezium.connector.mysql.MySqlConnector",
    "type": "source",
    "version": "2.0.0.Final"
  }
]
```

Reply:
right. let's do this. If the call to conector-plugins api fails due to some reason(this is new but believe me we have seen this happen!) OR if api succeeds and version is none:
- fall back to old behavior .i.e older version (just for backward compatibility)